src/main.ts

```diff
@@ -35,25 +36,29 @@ if (program['generator'] && program['framework'] && program['input'] && program[
 
 
 function create(generator: Generator, framework: Framework, inputPath: string, outputPath: string) {
-  if (generator === 'docjs' && framework === 'react') {
-    selectedParser('docjs', inputPath, outputPath);
+  if (generator === 'typedoc' && framework === 'react') {
```

Comment:
this check doesn't make any sense since it just passes the input data

Reply:
Refactored.